src/pipeline/src/manager/table.rs

```diff
@@ -49,6 +49,8 @@ use crate::error::{
 use crate::etl::transform::GreptimeTransformer;
 use crate::etl::{parse, Content, Pipeline};
 
+/// Pipeline version. string formatted as iso8601 timestamp with nanosecond precision
+/// if the version is None, it means the latest version of the pipeline
```

Comment:
We don't always format the timestamp as an iso8061 string.
```suggestion
/// Pipeline version. An optional timestamp with nanosecond precision.
/// If the version is None, it means the latest version of the pipeline.
```

Reply:
Added more detailed documentation on PipelineVersion.